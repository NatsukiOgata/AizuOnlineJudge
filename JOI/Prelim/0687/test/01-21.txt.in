61
994 1023 1007 1028 1026 1031 997 1015 995 1037 1011 1041 1032 1043 1019 1024 1047 992 1012 1022 1014 1003 1046 1034 1049 1021 1033 993 1030 1035 1051 1036 1027 990 1005 1009 999 1000 1017 1048 1002 1044 1016 1050 1025 1013 1020 996 1001 991 998 1008 1018 1006 1045 1042 1010 1004 1039 1038 1040
