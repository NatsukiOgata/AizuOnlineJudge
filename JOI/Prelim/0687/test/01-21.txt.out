30654
30537
